tests/Integrations/CLI/NoAutoloader/NoAutoloaderTestCase.php

```diff
@@ -0,0 +1,35 @@
+<?php
```

Comment:
Sorry for all the nit-picks about moving files around. Could we move this one to `tests/Integrations/CLI/Custom/NotAutoloaded` to be consistent with the autolaoded test? :)